content/cloud/pricing-and-billing.md

```diff
@@ -73,9 +73,9 @@ We hope you do not have a problem, but if you do as a Growth or Enterprise Plan
 **How do I report a problem with Fermyon Cloud?**
 We hope you do not have a problem, but if you do as a Growth or Enterprise Plan user, please contact [support@fermyon.com](mailto:support@fermyon.com). For our Starter users, we recommend reaching out to our [Discord community](https://discord.gg/AAFNfS7NGf) for assistance. 
 
-**What support is offered with Gthe Growth Plan?**
+**What support is offered with the Growth Plan?**
 
-We offer support with a one business day response time via email and a best-effort response time via online chat Monday through Friday. In addition, we offer a vibrant [Discord community](https://discord.gg/AAFNfS7NGf) where common technical questions are often answered faster than one business day, including occasional weekend answers.
+We offer support with a one business day response time via email and a best-effort response time via online chat Monday through Friday from 8am to 6pm Eastern Time Zone. In addition, we offer a vibrant [Discord community](https://discord.gg/AAFNfS7NGf) where common technical questions are often answered faster than one business day, including occasional weekend answers.
```

Comment:
Maybe add the Time zone abbreviation to make sure there's no confusion (ET): https://www.timeanddate.com/time/zones/et